prisma/migrations/20220715134600_init/migration.sql

```diff
@@ -0,0 +1,73 @@
+-- CreateTable
+CREATE TABLE `PaybuttonAddress` (
```

Comment:
Is it too ambiguous to just use `Address` ?

We're thinking that we'll eventually support other buttons like Cashtab's and not just our own.

Reply:
I thought about making this change, but didn't went through with it because I though it would be too time consuming and add a lot more changes to a PR that is already not that small. I can do it in another PR tho.

Reply:
Ok